### PR TITLE
Add QEMU support for RHEL 9

### PR DESCRIPTION
**Pull request by ahreehong**
## Change description
This PR adds RHEL 9 support for QEMU.

### CODE REVIEW

**Comment by mboersma** [comment]
> I thought the `/lgtm` command also handled `ok-to-test`

It really should. 👍🏻

**Comment by AverageMarcus** [comment]
```
open /home/prow/go/src/sigs.k8s.io/image-builder/images/capi/packer/ova/rhel-9.json: no such file or directory
```
😕  Not sure why the tests are failing on the OVA. That's not changed in this PR.

/test pull-packer-validate

**Comment by drew-viles** [comment]
I **think** OVA dynamically generates the build targets - https://github.com/kubernetes-sigs/image-builder/blob/2119419bf664d639a8cfb6e1f80e7516232dd41b/images/capi/Makefile#L393

File: images/capi/Makefile [hunk 1]
```diff
@@ -318,7 +318,7 @@ PACKER_POWERVS_NODE_FLAGS := $(foreach f,$(abspath $(COMMON_POWERVS_VAR_FILES)),
 CENTOS_VERSIONS			:=	centos-7
 FLATCAR_VERSIONS		:=	flatcar
 PHOTON_VERSIONS			:=	photon-3 photon-4 photon-5
-RHEL_VERSIONS			:=	rhel-7 rhel-8
+RHEL_VERSIONS			:=	rhel-7 rhel-8 rhel-9
```
Comment (AverageMarcus) [inline images/capi/Makefile:321]: Undoing this change should fix the tests
```suggestion
RHEL_VERSIONS			:=	rhel-7 rhel-8
```

**Comment by AverageMarcus** [comment]
Looks good to me!

@ahreehong would you mind squashing the commits please?

**Review by mboersma — APPROVED** [review]
/lgtm
/approve

**Pull request merged**